backend/ibutsu_server/tasks/importers.py

```diff
@@ -94,8 +99,19 @@ def run_junit_import(import_):
                 "tests": ts.get("tests"),
             },
         }
+
         if import_record.data.get("project_id"):
             run_dict["project_id"] = import_record.data["project_id"]
+
+        metadata = None
+        if import_record.data.get("metadata"):
+            # metadata is expected to be a json dict
+            metadata = import_record.data["metadata"]
+            run_dict["data"] = metadata
```

Comment:
Just verifying, the `update` is not needed because run metadata will always be empty for a junit xml right?

Reply:
Right, data doesn't actually exist in the run_dict at this point. I think on the normal junit upload it would've not existed at all before.